server/service/src/programs/patient/patient_updated.rs

```diff
@@ -35,7 +61,7 @@ pub fn patient_document_updated(
         r#type: NameType::Patient,
         is_customer: true,
         is_supplier: false,
-        supplying_store_id: Some(store_id.to_string()),
+        supplying_store_id: None,
```

Comment:
How would the patient's home store be saved if they're created in remote? 🤔

Reply:
Good point, I create a new PR where I take over values from the existing name row!